modules/auxiliary/admin/edirectory/edirectory_dhost_cookie.rb

```diff
@@ -22,7 +22,6 @@ def initialize(info = {})
       },
       'References'     =>
```

Comment:
Good eye. It seems to load fine, so I didn't try to change it. Let me know how you want it :-)